packages/react-async/README.md

```diff
@@ -173,6 +173,37 @@ const MyModalComponent = createAsyncComponent({
 });
 ```
 
+#### When to use MyComponent.Preload, MyComponent.Prefetch and MyComponent.KeepFresh
+
+```tsx
+<MyComponent.Preload />
+```
+
+Will only be required when you purposely want to control the pre-loading.
+
+_For example_
+
+```tsx
+const MyComponent = createAsyncComponent({
+  load: () => import(/* webpackChunkName: 'MyComponent' */ './MyComponent'),
+  usePreload: () => usePreload(MyQuery),
+});
+
+<MyComponent.Preload />;
+```
+
+When relying on the default behaviour we do not need to render `MyComponent.Preload`
```

Comment:
I would also mention the `defer` property and the fact that it is ignored / overwritten by the `.Preload` / `.Prefetch` components.